free-2.md

```diff
@@ -58,12 +176,25 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+To make the system really useable, a large amount of time would need to be spent
+on the semantics, unfortunately. If everything could be handled via shell
+escapes—especially if shell escapes were made to look like part of the
+language—then writing the language would be much easier but using it would
+require more external tools.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Getting the semantics of this project up to an easy-to-use level is ambitious.
+Building the language itself is probably not particularly ambitious, but that’s
+difficult to judge. The language can be extended pretty far, but it’s hard to
+tell how much of that would be library vs. DSL, and how much would be complex
+semantics vs. language design.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This could be a good idea for a project because it would produce something
+useful, and because it would be a good DSL. It might not be a good idea for a
+project if it turned out to be too semantics-heavy.
```

Comment:
This is a really interesting project idea. I agree that there might be a temptation to focus too much on semantics; but I think there will probably be a few language-design surprises along the way, too. In general, this idea would work well for a project. It's a good idea, you've thought it out, it's is reasonably scoped,  but it also has room to grow.